lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
they are forced in the browser too https://github.com/feross/instant.io/blob/c8924fe4cea665c938911edf5b7efaa2c4b1e9ab/client/index.js#L15

Only place where they aren't forced is webtorrent-cli, because it doesn't support webrtc anyway.

> Thought the default trackers will be added to the torrents anyway (see announceList).

They will be added to new torrents you create with createTorrent without giving a list of trackers manually. But they won't be added to existing torrents you download (/seed), which is the reason why this global.WEBTORRENT_ANNOUNCE exists